src/bento/components/bento-mega-menu/1.0/component/BentoItem.tsx

```diff
@@ -0,0 +1,63 @@
+import {Shim} from '#bento/components/bento-mega-menu/1.0/component/Shim';
+
+import * as Preact from '#preact';
+import {useLayoutEffect, useRef} from '#preact';
+import {useAttributeObserver} from '#preact/hooks/useMutationObserver';
+import {FC} from '#preact/types';
+
+import {Content} from './Content';
+import {Item, useMegaMenuItem} from './Item';
+import {Title} from './Title';
+
+export const BentoItem: FC = ({children}) => {
+  return (
+    <Item>
+      <SlottedDomWrapper>{children}</SlottedDomWrapper>
+    </Item>
+  );
+};
+
+/**
+ * Renders the slot, and uses a bunch of shims to control the slotted (light DOM) elements
+ */
+const SlottedDomWrapper: FC = ({children: slot}) => {
+  const {actions, isOpen} = useMegaMenuItem();
+
+  const ref = useRef<HTMLDivElement>(null);
+
+  // Capture all the Light DOM elements:
+  const sectionRef = useRef<HTMLElement>(null);
+  const titleRef = useRef<HTMLElement>(null);
+  const contentsRef = useRef<HTMLElement>(null);
+  useLayoutEffect(() => {
+    const slot = ref.current!.querySelector('slot')!;
```

Comment:
Nitpick - I got a little bit confused that both the children prop and this constant are named "slot".